src/connectedk8s/setup.py

```diff
@@ -37,6 +37,7 @@
 DEPENDENCIES = [
     'kubernetes==11.0.0',
     'pycryptodome==3.9.8',
+    'packaging'
```

Comment:
In fact, I see that the dependency of `packaging>=20.9,<22.0` has been added in `azure-cli-core`. Why do you need to add it here?
https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/setup.py#L57

In addition, don't we need to pin the version of `packaging`?

Reply:
azure-cli-core 2.16.0 doesn't have "packaging" , due to which it is unable to pull it

Reply:
This is because the CLI version used by the user is too old. Therefore, we recommend that the user upgrade the version used by CLI instead of adding duplicate dependencies in the CLI extension. Because adding duplicate dependencies can easily lead to package conflicts

Reply:
@zhoxing-ms is absolutely right. I really don't like the idea of adding `packaging` dependency because customer is using Azure CLI 2.16.0 which is just too old!